go.mod

```diff
@@ -241,6 +241,11 @@ require (
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
+// This is copied from the go.mod file in github.com/lxc/lxd
+// It is needed to avoid this error when running go list -m
+// go: google.golang.org/grpc/naming@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
+replace google.golang.org/grpc/naming => google.golang.org/grpc v1.29.1
```

Comment:
Does this mean we'll be pulling in v1.29 of grpc? If not, how does grpc/naming work? (Because that was removed in grpc v1.30.)